### ISSUE TITLE
Platform (dart:io) usage in Common testWidgets breaks Web

### ISSUE THREAD

**Issue by desmond206x** [leancodepl/patrol#1577]
### Steps to reproduce

Run this test

integration_test.dart
```
patrolTest('login with patrol', (PatrolTester $) async {
  await $.pumpWidgetAndSettle(AwesomeApp());

  await $(#emailTextField).enterText('charlie@root.me');
});
```
with
```
flutter drive --driver=test_driver/integration_test.dart --target=integration_test/integration_test.dart -d chrome
```

### Actual results

The fix from me for the enterText on Web was released with 2.0.2, but your bugfix for 2.0.4 (https://github.com/leancodepl/patrol/commit/c16b4c7f9b205dd7512fbe2c79fccd00a8c51623) broke the web compatibility again.

Very similar to this issue: https://github.com/leancodepl/patrol/issues/1457

dart:io is not supported on web and with your change we can't usw patrol for our web tests.

I know you don't support web officially, but I would like to fix this again, so we can use patrol again. 

### Logs

<details>
<summary>Logs</summary>

```
<!-- Replace this line with your logs. Do not remove the backticks! -->
```

</details>

### Patrol version

patrol: 2.0.4

### Patrol Doctor output

<details>
<summary>Patrol Doctor output</summary>

```
<!-- Replace this line with your logs. Do not remove the backticks! -->
```

</details>

### Flutter Doctor output

<details>
<summary>Flutter Doctor output</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.3.9, on Microsoft Windows [Version 10.0.19045.2846], locale de-DE)
[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[√] Chrome - develop for the web
[!] Visual Studio - develop for Windows (Visual Studio Professional 2022 17.0.6)
    X Visual Studio is missing necessary components. Please re-run the Visual Studio installer for the "Desktop development with C++" workload, and include these components:
        MSVC v142 - VS 2019 C++ x64/x86 build tools
         - If there are multiple build tool versions available, install the latest
        C++ CMake tools for Windows
        Windows 10 SDK
[√] Android Studio (version 2020.3)
[√] VS Code, 64-bit edition (version 1.79.2)
[√] Connected device (3 available)
[√] HTTP Host Availability

! Doctor found issues in 1 category.
```

</details>

**Comment by jBorkowska** [comment]
Hi @desmond206x, thanks for the issue. Our main developer is on vacation right now, but we will look into this problem once he's back

**Comment by bartekpacia** [comment]
Hi @desmond206x, the fix is available in patrol v2.2.1
